backend/game/engine/board.py

```diff
@@ -0,0 +1,35 @@
+
+import game.dice.dice_resolver as dice_resolver
+from game.engine.player_queue import GamePlayers
+import game.values.constants as constants
+import game.values.status as status
+
+
+class BoardGame():
+
+    def __init__(self):
+        self.players = GamePlayers()
+        self.STATUS = status.SETUP
+        self.WINNER = None
+
+    def start_game(self):
+        self.players.set_player_order()
+        self.STATUS = status.ACTIVE
+
+    def check_if_winner(self, player):
```

Comment:
You can also win by being last player alive

Reply:
Gerardo resolved this issue by adding this logic.